dbms/src/Encryption/RateLimiter.cpp

```diff
@@ -292,42 +292,22 @@ ReadLimiter::ReadLimiter(
     std::function<Int64()> getIOStatistic_,
     Int64 rate_limit_per_sec_,
     LimiterType type_,
-    Int64 get_io_stat_period_us,
     UInt64 refill_period_ms_)
     : WriteLimiter(rate_limit_per_sec_, type_, refill_period_ms_)
-    , getIOStatistic(std::move(getIOStatistic_))
-    , last_stat_bytes(getIOStatistic())
-    , last_stat_time(now())
+    , get_io_statistic(std::move(getIOStatistic_))
```

Comment:
I suggest changing the name of `get_io_statistic` and `getIOStatistic_`, such as `get_read_bytes`.